examples/examples_nodejs_test.go

```diff
@@ -265,14 +265,29 @@ func TestAccDurableFunctions(t *testing.T) {
 }
 
 func TestAccEventgrid(t *testing.T) {
-	// TODO: Fix this test.
-	t.Skip("This test fails to compile. Having difficulty resolving locally. See: https://github.com/pulumi/pulumi-azure/issues/1036")
 	test := getJSBaseOptions(t).
 		With(integration.ProgramTestOptions{
 			Dir:                      filepath.Join(getCwd(t), "eventgrid"),
 			RunUpdateTest:            false,
 			AllowEmptyPreviewChanges: true,
 			AllowEmptyUpdateChanges:  true,
+
+			// Getting the following diff:
+			//     "diffs": [
+			//         "webhookEndpoint"
+			//     ],
+			//     "detailedDiff": {
+			//         "webhookEndpoint.maxEventsPerBatch": {
+			//             "kind": "DELETE"
+			//         },
+			//         "webhookEndpoint.preferredBatchSizeInKilobytes": {
+			//             "kind": "DELETE"
+			//         }
+			//     }
+			// even though the webhook is simply created as
+			//     webhookEndpoint: { url: liveUrl }
+			// in eventgrid/zMixins.ts.
+			ExpectRefreshChanges: true,
```

Comment:
Do we have a tracking issue for the refresh changes (assuming this is something we want to eventually fix)

Reply:
I don't think this will make the cut to be fixed TBH